packages/shared-ux/page/kibana_no_data/BUILD.bazel

```diff
@@ -0,0 +1,148 @@
+load("@npm//@bazel/typescript:index.bzl", "ts_config")
+load("@build_bazel_rules_nodejs//:index.bzl", "js_library")
+load("//src/dev/bazel:index.bzl", "jsts_transpiler", "pkg_npm", "pkg_npm_types", "ts_project")
+
+PKG_DIRNAME = "kibana_no_data"
+PKG_REQUIRE_NAME = "@kbn/shared-ux-page-kibana-no-data"
```

Comment:
I know we are not 100% on naming conventions yet, but would this not be better as `kibana-page-no-data`?